pkg/gps/GpsMessage.php

```diff
@@ -181,11 +180,17 @@ public function setNativeMessage(?GoogleMessage $message = null): void
         $this->nativeMessage = $message;
     }
 
+    /**
+     * @deprecated This will be removed in the V1, use headers instead
```

Comment:
nah, I think we can break it right away. I don think there are many people that started using it.

Reply:
thanks just drop the last commit.